testsuite/tests/typing-layouts-products/basics_implicit_unboxed_records.ml

```diff
@@ -136,7 +136,9 @@ Line 4, characters 2-7:
 4 |   left'
       ^^^^^
 Error: This value is "local"
-       because it is from the allocation (at Line 2, characters 10-25) containing a value
+       because it is the field "left" of the record at Line 3, characters 6-25
+       which is "local"
+       because it is allocated at Line 2, characters 10-25 with content
```

Comment:
Note that some "containing" relation hint is stil missing: It should mention that `t` is "local" to the parent region because it contains field `left` which is `local` to the parent region (and points to L131).

I will investigate; but also fine to merge as it is.